src/Model/ResourceField.php

```diff
@@ -58,6 +60,15 @@ public function getCMSFields()
             $fields->removeByName('Type');
             $fields->dataFieldByName('ReadableName')
                 ->setAttribute('placeholder', $this->Name);
+            $orderField = NumericField::create('Order')
+                ->setTitle(i18n::_t(__CLASS__ . '.OrderLabel', 'Presented order'))
```

Comment:
As mentioned in the other PR I feel like UPPER_SNAKE_CASE is more common in our code. Maybe I'm delusional and we're just being super inconsistent everywhere 😆

Reply:
A little bit. As I mentioned on the other PR - the inconsistency makes it a bit of a debate as to the style chosen, as there's no "enough" usage of a single style to prefer it over another I think.

We have used CamelCase lower down for `ResultConditions` but that's OK I'll update it too :)
(I kind prefer the upper case, for no good reason).